src/pwr.rs

```diff
@@ -220,6 +220,48 @@ macro_rules! d3cr {
     };
 }
 
+/// Returns the voltage scale at the current moment
+pub(crate) fn current_vos() -> VoltageScale {
+    // NOTE(unsafe): Read-only access
+    #[cfg(all(
+        feature = "revision_v",
+        any(feature = "rm0433", feature = "rm0399")
+    ))]
+    if unsafe { (*SYSCFG::ptr()).pwrcr.read().oden().bit_is_set() } {
+        return VoltageScale::Scale0;
+    }
+
+    // NOTE(unsafe): Read-only access
+    let d3cr = unsafe { d3cr!(*PWR::ptr()).read() };
```

Comment:
Doesn't clippy complain about dead code if the cfg above is active?

Reply:
This code is still reachable at runtime even in the cfg is active, since the return is conditional